### PR TITLE
Modify apimachinery,apps,instrumentation tests to import framework/log

### PR DESCRIPTION
**Pull request by figo**
This is part of the transition to using framework/log instead
of the Logf inside the framework package. This will help with
import size/cycles when importing the framework or subpackages

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
#77359

```release-note
N/A
```

### CODE REVIEW

File: test/e2e/instrumentation/monitoring/stackdriver.go [hunk 1]
```diff
@@ -27,8 +27,9 @@ import (
 
 	"github.com/onsi/ginkgo"
 	"k8s.io/apimachinery/pkg/util/wait"
-	"k8s.io/kubernetes/test/e2e/common"
+	e2ecommon "k8s.io/kubernetes/test/e2e/common"
```
Comment (andrewsykim) [inline test/e2e/instrumentation/monitoring/stackdriver.go]: Generally agree with this change but can we undo this one and `e2emetrics`, think it's better to keep the PR only changing e2elog imports.

Comment (figo) [reply test/e2e/instrumentation/monitoring/stackdriver.go]: fixed now

**Comment by figo** [comment]
a new change on test/e2e/apimachinery/chunking.go checked-in today, i am going to do a force push to resolve the conflict, appreciate if you can add lgtm again @oomichi  @andrewsykim @timothysc

**Comment by oomichi** [comment]
@figo Thanks for maintaining this.

/lgtm

**Review by timothysc — APPROVED** [review]
/lgtm 
/approve

**Comment by oomichi** [comment]
@figo Could you rebase this again, sorry about that.

**Comment by figo** [comment]
> @figo Could you rebase this again, sorry about that.

thx, rebase done, PTAL @oomichi

**Pull request merged**